db-models/listings.sql

```diff
@@ -0,0 +1,14 @@
+-- db-models/listings.sql
+CREATE TABLE listings (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,       -- Auto-incrementing ID
```

Comment:
I think we should change this to `listing_id` to keep things clear, especially once we start mixing in with other DB models